src/main/charts/confluence/templates/statefulset-synchrony.yaml

```diff
@@ -22,6 +22,20 @@ spec:
     spec:
       serviceAccountName: {{ include "confluence.serviceAccountName" . }}
       terminationGracePeriodSeconds: {{ .Values.synchrony.shutdown.terminationGracePeriodSeconds }}
+      {{- if .Values.synchrony.securityContextEnabled }}
+      {{- with .Values.synchrony.securityContext }}
+      securityContext:
+        {{/* this condition will be removed in v2.0.0 */}}
```

Comment:
I don't think we need the condition here. We never had this defined for Synchrony statefulset so providing backward compatibility is not necessary.

Reply:
Thanks @nanux I wasn't quite sure why this condition was there, so decided not to delete it. Fixed now.